### DISCUSSION TITLE
In need of import 15 groups

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#14555] [How to]
Gents, I have been asked for importing into yetiforce system around 15.000 groups. Is there any safe way to do so?

**Comment by vovpff** [comment]
What is the group?

**Reply by waran70** [reply]
![image](https://user-images.githubusercontent.com/6852348/114813237-e62c6580-9ddb-11eb-8653-626c0842c3c1.png)
I guess he means this

**Reply by mariuszkrzaczkowski** [reply]
- Own script running on the database
- pay and we will add the import to the system
